### ISSUE TITLE
Update the Accessibility page to match accessibility standards.

### ISSUE THREAD

**Issue by joedolson** [WordPress/wporg-main-2022#494]
**Describe the bug**
The page at WordPress > About > Accessibility currently specifies WCAG 2.0 level AA, but the actual standards specify WCAG 2.1. 

Also be aware that I've opened a request to update the Accessibility standards to reflect WCAG 2.2; these changes should be coordinated.

**To Reproduce**
1. Go to https://wordpress.org/about/accessibility/
2. Go to https://developer.wordpress.org/coding-standards/wordpress-coding-standards/accessibility/
3. See that these documents don't match.

**Expected behavior**
Documents should match.

**Comment by StevenDufresne** [comment]
Confirming... [this page](https://wordpress.org/about/accessibility/) needs to be updated from `WCAG 2.0 AA` to `WCAG 2.1 AA` and in the future we'll need to update to `WCAG 2.2`, pending a request opened somewhere else?

If that's correct, can we link that pending request for now? Ideally we get a new ticket when that happens for tracking.

**Comment by joedolson** [comment]
Sorry; I fully intended to link that...

https://github.com/WordPress/wpcs-docs/pull/141

**Comment by joedolson** [comment]
Noting that the related WPCS doc has now been updated, so it's already time to adjust this to say 2.2. That was quick. ;)
